org.greenfoot.Greenfoot.json

```
@@ -0,0 +1,82 @@
+{
+  "app-id": "org.greenfoot.Greenfoot",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "21.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk11"],
+  "command": "org.greenfoot.Greenfoot",
+  "finish-args": [
+    "--env=PATH=/app/jre/bin:/usr/bin:/app/bin",
+    "--share=network",
+    "--socket=x11",
+    "--share=ipc",
+    "--socket=wayland",
+    "--device=dri",
+    "--socket=pulseaudio",
+    "--filesystem=host",
+    "--filesystem=xdg-run/dconf",
+    "--filesystem=~/.config/dconf:ro",
+    "--talk-name=ca.desrt.dconf",
+    "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```

```suggestion
    "--filesystem=xdg-run/dconf",
    "--filesystem=~/.config/dconf:ro",
    "--talk-name=ca.desrt.dconf",
    "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```

These are not needed with modern runtimes.

---

Ok